.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
 node_js:
-  - "stable"
+  - '11.11.0'
```

Comment:
can we use an LTS node version here?

Reply:
Yes. I have updated this PR @Japhethca 
Thanks.